pywps/cli.py

```diff
@@ -0,0 +1,124 @@
+###########################################################
+# Demo WPS service for testing and debugging.
+#
+# See the werkzeug documentation on how to use the debugger:
+# http://werkzeug.pocoo.org/docs/0.12/debug/
+###########################################################
+
+import os
+import sys
+import click
+import signal
+import threading
+from werkzeug.serving import run_simple
+from pywps.app.Service import Service
+from pywps import configuration
+from pywps.queue import JobQueueService
+
+from alembic.config import Config
+from alembic import command
+
+from urllib.parse import urlparse
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+def get_host():
+    url = configuration.get_config_value('server', 'url')
+    url = url or 'http://localhost:5000/wps'
+
+    click.echo("starting WPS service on {}".format(url))
+
+    parsed_url = urlparse(url)
+    if ':' in parsed_url.netloc:
+        host, port = parsed_url.netloc.split(':')
+        port = int(port)
+    else:
+        host = parsed_url.netloc
+        port = 80
+    return host, port
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+@click.version_option()
+@click.option('--config', '-c', metavar='PATH', help='path to pywps configuration file.')
+def cli(config):
+    """Command line to start/stop a PyWPS service.
+
+    Do not use this service in a production environment.
+    It's intended to be running in a test environment only!
+    For more documentation, visit http://pywps.org/doc
+    """
+    if config:
+        os.environ['PYWPS_CFG'] = config
+
+
+@cli.command()
+@click.option('--bind-host', '-b', metavar='IP-ADDRESS', default='127.0.0.1',
+              help='IP address used to bind service.')
+@click.option('--no-jobqueue', '-n', is_flag=True,
+              help='Do not start job queue service.')
+def start(bind_host, no_jobqueue):
+    """Start PyWPS service.
+    This service is by default available at http://localhost:5000/wps
+    """
+    app = Service()
```

Comment:
made an update to allow overwrite of pywps config with env `PYWPS_CFG`:
https://github.com/geopython/pywps/pull/505/commits/1d0d3d86e193b848cb77465557ea40831f27740d

The migrate command is still using `PYWPS_CFG` and can not be overwritten.

It would possible to add multiple configs at the command line ... not sure if we want this.